src/main/java/org/homework/Option/DeleteOption.java

```diff
@@ -0,0 +1,13 @@
+package org.homework.Option;
+
+public class DeleteOption implements OptionsInterface{
+    @Override
+    public void process() {
+        outputView.printDelete(toDoRepository.deleteList(inputView.inputNumber()));
+    }
```

Comment:
어댑터패턴처럼 클래스의 결이 비슷한 존재에 한해서 한줄로 줄여쓰는게 가독성이 좋은 경우가 있습니다.
``` java 
BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
```

해당 코드는 입력 받고, 삭제하고, 출력하는 과정이 명확하게 나누어져 있기 때문에 
코드를 나누고 변수명을 선언하는 것이 가독성, 유지보수성, 명확성 등 측면에서 더 좋습니다. 

```suggestion

int deleteId = inputView.inputNumber();

boolean deleted = toDoRepository.deleteList(deleteId);

outputView.printDelete(deleted);

```